### PR TITLE
Add new Hacktoberfest features Urdu translations.

### PR DESCRIPTION
**Pull request by Toprun123**
Added the new translations for Urdu from the `en/messages.json` file. The the order of fields couldn't be matched so I left it as is. Could you please accept this before Hacktoberfest ends? @rugk

### CODE REVIEW

**Review by Toprun123 — COMMENTED** [review]
Added 8 new translations in urdu.

**Review by rugk — APPROVED** [review]
Ah thanks. You also were the initial contributor, so I have nobody else for review, so I'll merge this.

And you get one more PR for Hacktoberfest! :wink:

File: src/_locales/ur/messages.json
Comment (rugk) [inline src/_locales/ur/messages.json:190]: This is apparently missing a translation though hmm… cannot merge it as that, but I'll add a tag so it counts as accepted :wink:

**Review by rugk — CHANGES_REQUESTED** [review]

**Comment by rugk** [comment]
BTW what do you mean by:
> The the order of fields couldn't be matched so I left it as is. 

Which fields do you mean? If there are placeholders, you can usually just place them where you want (but there should be none in the new strings). Or do you mean the order in the JSON file? That does not really matter, but also looks fine for me, actually.

File: src/_locales/ur/messages.json
Comment (Toprun123) [reply src/_locales/ur/messages.json:190]: It's because i did not understand what the & symbol is doing.. Should i replace the entire text or leave the &Generate part (there are 5 of these that i missed.)

**Comment by Toprun123** [comment]
> Or do you mean the order in the JSON file? That does not really matter, but also looks fine for me, actually.

That's what I meant, the order in the JSON file.

File: src/_locales/ur/messages.json
Comment (Toprun123) [reply src/_locales/ur/messages.json]: I translated it all to Urdu (Maybe I should have left the word following the `&`?)

Comment (rugk) [reply src/_locales/ur/messages.json]: As for the ampersand (`&?) symbol, what this means [is explained here](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations). Basically it's for creating a keyboard hotkey.

**Review by rugk — APPROVED** [review]
As for the ampersand symbol see my comment above.
As for our language I am also not sure how these can be translated, but here is what the docs say, which may be relevant:
> In some localized versions of Firefox (Japanese and Chinese), the access key is surrounded by parentheses and appended to the menu label, unless the menu title itself already ends with the access key ("toolkit(&K)" for example). For more details, see [Firefox bug 1647373](https://bugzil.la/1647373).

Actually, you could also [try out the development version](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#coding) and report back whether it/everything works/you like it or what may still be adjusted, so I can do this before the next release. In case there is something to be done… :smile: 

Just so we know the translation works good in the browser when it is actually used.

Anyway, I'll merge this for now, if you need to adjust/want to adjust something more, please just create a new PR. And thanks again for the translation!

**Pull request merged**
